tools/run_tests/artifact_targets.py

```diff
@@ -62,7 +62,7 @@ def create_jobspec(name, cmdline, environ=None, shell=False,
           cmdline=cmdline,
           environ=environ,
           shortname='build_artifact.%s' % (name),
-          timeout_seconds=10*60,
+          timeout_seconds=15*60,
```

Comment:
I lifted this to 30mins which ought be enough for everyone. If it takes more time than that, I wouldn't be pretty sad (and it would be worth fixing).